conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -50,5 +52,11 @@ conda build /recipe_root --quiet || exit 1
 {%- endfor -%}
 {%- endblock -%}
 
+touch /feedstock_root/build_artefacts/conda-forge-build-done
 EOF
 
```

Comment:
Missed that we did not include another empty line below. As a result, this means the terminal newline would be dropped from this file. Fixing it in PR ( https://github.com/conda-forge/conda-smithy/pull/459 ).